docs/detections/session-view.asciidoc

```diff
@@ -84,4 +93,35 @@ Session View includes an additional badge not pictured above:
++
+[role="screenshot"]
+image::images/session-view-output-badge.png[The Output badge,width=80%,height=80%]
+
+[[session-view-output]]
+[discrete]
+=== Terminal output view UI
+
+beta::[]
+
+Terminal output view helps provide a more complete understanding of commands executed by users or adversaries, and assists with auditing and compliance objectives. It allows you to review a session's TTY output, in the context of the commands which generated it. It includes both TTY output, and commands entered into a shell as typed — provided the shell was in echo mode.
+
+To enable Terminal output data capture: go to *Manage* -> *Policies*
+and edit one or more of your {elastic-defend} integration policies. On the *Policy settings* tab,
+scroll down to the Linux event collection section near the bottom of the page
+and turn on *Include session data* and *Capture terminal output*. You can configure several additional settings by clicking *Advanced settings* at the bottom of the page:
+
+* Select the maximum number of kilobytes of output to record from a single process by modifying `linux.advanced.tty_io.max_kilobytes_per_process`. Defaults to 512 KB. For example, with the default value, a process' output will not be recorded after the 512th KB.
+* Select the maximum number of kilobytes of output to send to {es} as a single event by modifying `linux.advanced.tty_io.max_kilobytes_per_event`. Defaults to 512 KB. For example, with the default value, the maximum amount of output that will be sent to {es} as a single event is 512 KB. Additional data will be captured as a new event.
+* Select the maximum duration in seconds for which output is batched by modifying `linux.advanced.tty_io.max_event_interval_seconds`. Defaults to 30 seconds. For example, with the default value, any recorded output will be sent to {es} every 30 seconds (unless it first exceeds the `max_kilobytes_per_event` value, in which case it might be sent sooner).
+
+[role="screenshot"]
+image::images/session-view-output-viewer.png[Terminal output view,width=100%,height=100%]
+
+1. The search bar. Use to find and highlight search terms within the current session.
+The buttons on its right side allow you to jump through search results.
+2. The right-side scroll bar. Use along with the bottom-side scroll bar to navigate output data that doesn't fit on a single screen.
+3. The playback controls: *Start*, *Prev*, *Play*, *Next*, *Last*, and the progress bar. Use to advance or rewind the session's commands and output. Click anywhere on the progress bar to jump to that part of the session. The marks on the bar represent processes that generated output, click them or click the *Prev* and *Next* buttons to skip between processes.
+4. The *Fit screen*, *Zoom in*, and *Zoom out* buttons. Use to adjust the text size. Use Session view's *Fullscreen* button (located next to the *Close session viewer* button) to improve the display of output that doesn't easily fit, especially for graphical programs like `vim`.
```

Comment:
I didn't think the last sentence was necessary, but feel free to veto me on this one. 
```suggestion
4. The *Fit screen*, *Zoom in*, and *Zoom out* buttons. Use to adjust the text size. Use Session view's *Fullscreen* button (located next to the *Close session viewer* button) to expand the display output.
```